01-getting-started/003-ecr-setup.md

```diff
@@ -58,14 +58,14 @@ provider "aws" {
 module "ecr-repo" {
   source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"
```

Comment:
can you please use ref=1.0 and maybe add a note about checking available releases?

Reply:
yes good point @razvan-moj